### PR TITLE
[fix](nereids) fix The children format needs to be [WhenClause+, DefaultValue?]

### PR DESCRIPTION
**Pull request by 924060929**
## Proposed changes

fix The children format needs to be [WhenClause+, DefaultValue?]

### CODE REVIEW

**Comment by 924060929** [comment]
run buildall

**Comment by 924060929** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/monitor/jvm/JvmService.java [hunk 1]
```diff
@@ -38,20 +38,16 @@ public class JvmService {
 
     private final JvmInfo jvmInfo;
 
-    private JvmStats jvmStats;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/monitor/jvm/JvmService.java:41]: what's this?

**Comment by 924060929** [comment]
run buildall

**Comment by 924060929** [comment]
run buildall

**Comment by 924060929** [comment]
run buildall

**Comment by 924060929** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**